setup.py

```diff
@@ -53,7 +53,8 @@ def get_version(filename):
         'importlib_resources~=1.0.2;python_version=="3.4"',
         'importlib_resources~=1.4;python_version<"3.7" and python_version!="3.4"',
         'networkx~=2.4;python_version>="3.5"',
-        'networkx<=2.2;python_version<"3.5"'
+        'networkx<=2.2;python_version<"3.5"',
+        'junit-xml~=1.9',
```

Comment:
@miparnisari need to vet [this new dependency](https://pypi.org/project/junit-xml/)

Reply:
License info for `junit-xml`
- Freely Distributable
- OSI Approved :: MIT License